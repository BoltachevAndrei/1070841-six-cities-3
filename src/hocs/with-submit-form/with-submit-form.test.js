import React from 'react';
import renderer from 'react-test-renderer';
import withSubmitForm from './with-submit-form.js';

const id = 1;

const isPostingComment = false;

it(`withSubmitForm renders correctly`, () => {
  const tree = renderer
    .create(
        <withSubmitForm
          id={id}
          isPostingComment={isPostingComment}
          onSubmit={() => {}}
        />
    )
    .toJSON();
  expect(tree).toMatchSnapshot();
});
