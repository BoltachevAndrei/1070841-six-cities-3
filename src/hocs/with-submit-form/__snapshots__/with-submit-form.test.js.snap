// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`withSubmitForm renders correctly 1`] = `
<withSubmitForm
  id={1}
  isPostingComment={false}
  onSubmit={[Function]}
/>
`;
